node/src/commands/destroy.js

```diff
@@ -25,7 +25,10 @@ const destroy = async options => {
     await deployUtils.updateEnvironment(environment, { requiresApproval: requiresApproval });
   }
 
-  const deployment = await deployUtils.destroyEnvironment(environment);
+  const params = removeEmptyValuesFromObj({
+    [SKIP_STATE_REFRESH]: options[SKIP_STATE_REFRESH]
+  });
```

Comment:
why bother? what's the harm in undefined here? it seems the api wii manage, so it is kind of redundant, no? no need for `removeEmprtyValuesFromObj` IMO

Reply:
In normal usage, you don't want to see prints from the cli such as: sending `param: undefined` to api-route

Unless the user explicitly used the param, it should not be sent.